src/main/resources/data/minecolonies/researches/civilian/1_firstaid.json

```diff
@@ -0,0 +1,27 @@
+{
+  "id": "firstaid",
+  "name": "First Aid",
+  "branch": "civilian",
+  "requiredUniversityLevel": 1,
+  "requiredItems":
+  [
+  {
+    "itemName": "minecraft:hay_block",
+    "itemCount": 8
+  }
+  ],
+  "requiredBuildings":
```

Comment:
Thinking about making a modpack, I can see wanting to have required RESEARCHES (beyond parent) in addition to buildings and items.

Reply:
That was definitely an intent here, though making it clear enough from the user perspective without significant modifications to the UniversityWindow stuff was a little daunting.

Adding parsing support for additional requiredResearch is definitely a thing I can do now.

Reply:
That sounds good. Let's get the JSON right, and if there is later UI stuff, we can do that separately, or I can help do it here. There are options. :)